ComponentKit/Components/CKButtonComponent.mm

```diff
@@ -95,6 +95,10 @@ + (instancetype)newWithTitles:(const std::unordered_map<UIControlState, NSString
                    attributes:(const CKViewComponentAttributeValueMap &)passedAttributes
    accessibilityConfiguration:(CKButtonComponentAccessibilityConfiguration)accessibilityConfiguration
 {
+  const auto titles = titlesParam.take();
```

Comment:
eeeek, not sure i love this. what's the reason for not just having automatic conversion?